engine/samples/input/assets/sample.bind

```diff
@@ -0,0 +1,52 @@
+{
+    "action": {
```

Comment:
I spent some time confused does to this not being plural, maybe change it to `actions`